src/components/Registration/NatureOfBusiness.vue

```diff
@@ -0,0 +1,224 @@
+<template>
+  <div
+    id="nature-of-business"
+    class="ml-n6 pl-6 rounded-0"
+    :class="{ 'invalid-section': showErrors && !isFormValid }"
+  >
+    <v-row>
+      <v-col cols="12" sm="2">
+        <label>
+          <strong>Nature of Business</strong>
+        </label>
+      </v-col>
+
+      <v-col cols="12" sm="10">
+        <template v-if="state !== States.SUMMARY">
+          <p>
+            Enter one or more keywords that describe the primary nature of your business or enter
+            the six-digit NAICS code. Learn more by visiting the
+            <a
+              href="https://www.statcan.gc.ca/en/subjects/standard/naics/2022/v1/index"
+              target="_blank"
+              rel="noopener noreferrer"
+            >
+              <span>Statistics Canada website</span>
+            </a>
+            <v-icon small color="primary">mdi-open-in-new</v-icon>.
+          </p>
+
+          <v-text-field
+            filled
+            persistent-hint
+            autocomplete="chrome-off"
+            label="Keywords or Six-Digit NAICS"
+            hint="Example: landscaping, grocery, care repair, etc."
+            v-model="searchField"
+            :name="Math.random()"
```

Comment:
Was this related to some form of auto-complete? 
Sorry i can't recall the reasoning why this was done, although i know there was a reason!

Reply:
Yes, exactly. To disable auto-complete in Chrome, you need both `autocomplete="chrome-off"` and a unique element name, which this does.

Reply:
ah right, ty for the refresher!